components/application/activity-select.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Link from 'next/link' // TODO: Should be avoided
+import { Icon } from '@oacore/design'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './activity-select.css'
+
+const routes = [
+  {
+    path: '/',
+    icon: 'overview',
+    title: 'Overview',
+  },
+  {
+    path: '/data',
+    icon: 'data',
+    title: 'Data',
+  },
+  {
+    path: '/statistics',
+    icon: 'statistics',
+    title: 'Statistics',
+  },
+  {
+    path: '/plugins',
+    icon: 'plugins',
+    title: 'Plugins',
+  },
+  {
+    path: '/settings',
+    icon: 'settings',
+    title: 'Settings',
+  },
+]
+
+const ActivitySelect = React.memo(({ className }) => (
```

Comment:
For me, it's just a primary/secondary menu... I think `ActivitySelect` is not a good name for this component.

Reply:
Yes, it is. However, we should be more specific in naming to make clear what the product is.